merino-suggest/Cargo.toml

```
@@ -13,9 +13,12 @@ serde = { version = "1.0.125", features = ["derive"] }
 serde_json = "1.0"
 serde_with = "1.9.1"
 thiserror = "1.0"
+tracing-actix-web = "0.4.0-beta.9"
 fake = { version = "2.4", features = ["derive"] }
 rand = "0.8"
 tokio = { version = "1", features = ["time"] }
+uuid = "0.8.2"
```

If we keep this, we should add the v4 feature:

```suggestion
uuid = { version = "0.8.2", features = ["v4"] }
```

That feature enables the `new_v4()` method you're using. It is working "accidentally" now because `tracing-actix-web` also enables that feature, but this is a behavior of Cargo that is being changed in the 2021 edition.